### PR TITLE
changed the config file to take the choice

### PR DESCRIPTION
**Pull request by Hackmey**
**Notes for Reviewers**

This PR fixes ##10330

I think this is a user side issues while setting up the meshery. When i reinstalled meshery and properly set it up it worked.
I could not find any other root of this issue. 

However , an issue occured frequently while entering the choice , it used to terminate due to an unexpected newline.
So, I just changed that file --> mesheryctl\internal\cli\root\system\config.go

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ashparshp** [comment]
Thank you @Hackmey
Let's discuss this on the Meshery dev meeting. Add it as an agenda item to the [Meeting Minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk)., if you would :)

**Comment by vishalvivekm** [comment]
> Thank you @Hackmey Let's discuss this on the Meshery dev meeting. Add it as an agenda item to the [Meeting Minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk)., if you would :)

added // @Hackmey

File: mesheryctl/internal/cli/root/system/config.go [hunk 1]
```diff
@@ -406,7 +406,12 @@ func setToken() {
 		fmt.Print("Enter choice (number): ")
 		_, err = fmt.Scanf("%d", &choice)
 		if err != nil {
-			log.Fatalf("Error reading input:  %s", err.Error())
+			log.Warnf("Error reading input:  %s", err.Error())
+			log.Info("Let's try again. Please enter the choice (number):")
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/system/config.go:410]:
```suggestion
			log.Info("Let's try again. Please enter choose an option by number:")
```

File: mesheryctl/internal/cli/root/system/config.go [hunk 2]
```diff
@@ -406,7 +406,12 @@ func setToken() {
 		fmt.Print("Enter choice (number): ")
 		_, err = fmt.Scanf("%d", &choice)
 		if err != nil {
-			log.Fatalf("Error reading input:  %s", err.Error())
+			log.Warnf("Error reading input:  %s", err.Error())
+			log.Info("Let's try again. Please enter choose an option by number:")
+			_, err = fmt.Scanf("%d", &choice)
+			if err != nil {
+				log.Fatalf("Error reading input: %s", err.Error())
```
Comment (jameshorton2337) [inline mesheryctl/internal/cli/root/system/config.go:413]: Can we offer more description feedback to the user?

**Pull request closed without merging**